providers/implementations/ciphers/cipher_cts.c

```diff
@@ -48,19 +48,20 @@
 
 #include "e_os.h" /* strcasecmp */
 #include <openssl/core_names.h>
-#include <openssl/aes.h>
 #include "prov/ciphercommon.h"
 #include "internal/nelem.h"
-#include "cipher_aes_cts.h"
+#include "cipher_cts.h"
 
 /* The value assigned to 0 is the default */
 #define CTS_CS1 0
 #define CTS_CS2 1
 #define CTS_CS3 2
 
+#define CTS_BLOCK_SIZE 16
```

Comment:
This effectively limits this generic CTS implementation to block ciphers with 128 bit block size. Maybe add a comment above.